src/main/java/org/manifold/compiler/front/ExpressionEdge.java

```diff
@@ -30,8 +33,7 @@ public void setTarget(ExpressionVertex newTarget) {
 
   public ExpressionEdge(ExpressionVertex from, ExpressionVertex to) {
     if (from == null && to == null) {
-      throw new UndefinedBehaviourError(
-          "attempt to create a totally disconnected ExpressionEdge");
+      log.warn("attempt to create a totally disconnected ExpressionEdge");
```

Comment:
It's actually not an error any more, because there is a part of subgraph copying (and function elaboration, by extension) where we temporarily create totally disconnected edges and then connect them later on when the correct vertices are in place.